### DISCUSSION TITLE
Unable to import animate.css via webpack

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#12959] [Q&A] [answered]
```
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: The target selector was not found.
Use "@extend .animated !optional" to avoid this error.

   ╷
10 │         @extend .animated;
   │         ^^^^^^^^^^^^^^^^^
   ╵
```

I receive this above error while importing the animate.css in my scss file.

**Comment by alexander-akait** [comment, marked as answer]
Problem from `sass`, `node-sass` and `dart-sass` have different logic for `@extend`, you need open an issue in `animate.css` package, they should support `dart-sass`

**Reply by knightswatch3** [reply]
@alexander-akait This is compatibility issue with webpack 5 ? Because things worked well with my webpack version 3.

**Reply by alexander-akait** [reply]
The latest version  of `sass-loader` prefer `sass` (`dart-sass` over `node-sass`), before it was differently

**Reply by knightswatch3** [reply]
@alexander-akait I believe I am already setting the loader options as sass.
`{
            loader: "sass-loader",
            options: {
            implementation: require("sass"),
            sassOptions:{
                includePaths: config.client.sass_include_paths
              }
            }
          }`

But I am not sure what's causing this.

I've got the main.scss file which has all the imports:

`
@import "~animate.css"
@import "../../client_modules/**/*.scss";
 @import "../../view/**/*.scss";`

And I get the error stating unable to extend .animated.

**Reply by alexander-akait** [reply]
Please create simple reproducible test repo and I will look
